Scripts/GameManagement/GameManager.gd

```diff
@@ -19,6 +21,9 @@ func prepare_game():
 	rpc("initialize_world")
 	rpc("initialize_players")
 	rset("in_game", true)
+	var zone = world.get_node("BattleRoyaleMap/Zone")
+	connect("game_started", zone, "_on_Game_Started")
+	emit_signal("game_started")
```

Comment:
This should be part of the World initialization. I don't know if we necessarily need that signal. Especially if there will be other map where there is no zone (1v1, 3v3)

Reply:
The signal is needed to let the zone know that it can connect timers to each player (they need to be initialized to do that).